frontend/src/routes/FacilityBooking/ViewBookingDailyView/index.tsx

```diff
@@ -1,16 +1,15 @@
 import React, { useEffect } from 'react'
 import styled from 'styled-components'
-import { useHistory, useLocation } from 'react-router-dom'
+import { useHistory, useLocation, useParams } from 'react-router-dom'
 import { RootState } from '../../../store/types'
 import { useDispatch, useSelector } from 'react-redux'
 import LoadingSpin from '../../../components/LoadingSpin'
 import { PATHS } from '../../Routes'
 import ViewSection from '../../../components/FacilityBooking/ViewSection'
 import TopNavBarRevamp from '../../../components/TopNavBarRevamp'
-import { updateDailyView } from '../../../store/facilityBooking/action'
+import ButtonComponent from '../../../components/Button'
+import { setIsLoading, updateDailyView } from '../../../store/facilityBooking/action'
 import DailyViewDatesRow from '../../../components/FacilityBooking/DailyViewDatesRow'
-import { fetchFacilityNameFromID, setSelectedFacility } from '../../../store/facilityBooking/action'
-import { UnorderedListOutlined } from '@ant-design/icons'
 
 const HEADER_HEIGHT = '70px'
```

Comment:
should this be imported? since it is also in the create booking daily view file?

Reply:
might want to check which file should export/import it, im not too sure

Reply:
both pages import from the components folder